scripts/qa_crowbarsetup.sh

```diff
@@ -1587,6 +1587,14 @@ EOF
     # setup cloud repos for tftpboot and zypper
     onadmin_prepare_cloud_repos
     onadmin_add_cloud_repo
+    # FIXME: ugly hack - we need to prepare also SP1 Cloud repositories
+    # to correctly install SP1 nodes on SP2-enabled Cloud7
+    if iscloudver 7plus && [[ $want_sles12sp2 ]] ; then
+        unset want_sles12sp2
+        onadmin_prepare_cloud_repos
+        onadmin_add_cloud_repo
```

Comment:
IMHO the nicer way would be to leave the variable as is, but override the value for both lines thusly:
```bash
want_sles12sp2= onadmin_add_cloud_repo
```